### PR TITLE
Parameters of MOABB datasets

### LINKED ISSUE #364: Parameters of MOABB datasets
Hi all!
  It would be nice if we could pass some parameters when instantiating MOABB datasets through `braindecode.datasets.MOABBDataset`.
  For example, `Lee2019_MI` contains some test runs that can optionally be load with  `moabb.datasets.Lee2019_MI(test_runs=True)` but currently, we can not load them through `braindecode.datasets.MOABBDataset`.
  I can implement it if you believe it has its place in the library.

  Best,
  Pierre

### PR DESCRIPTION
**Pull request by PierreGtch**
Closes #364

### CODE REVIEW

**Comment by PierreGtch** [comment]
@agramfort I just fixed the style errors

File: braindecode/datasets/moabb.py [hunk 1]
```diff
@@ -72,13 +76,15 @@ def fetch_data_with_moabb(dataset_name, subject_ids):
         the name of a dataset included in moabb
     subject_ids: list(int) | int
         (list of) int of subject(s) to be fetched
-
+    dataset_kwargs: dict, optional
+        optional dictionary containing keyword arguments
+        to pass to the moabb dataset when instanciating it.
```
Comment (robintibor) [inline braindecode/datasets/moabb.py:81]: Is instantiate or instanciate correct spelling?

**Comment by robintibor** [comment]
thanks looks good, just wonder about one small spelling thing

**Comment by PierreGtch** [comment]
Thanks you're right, it's instantiating. I fixed it

Le 28/11/2021 à 22:00, robintibor a écrit :

File: braindecode/datasets/moabb.py [hunk 2]
```diff
@@ -72,13 +76,15 @@ def fetch_data_with_moabb(dataset_name, subject_ids):
         the name of a dataset included in moabb
     subject_ids: list(int) | int
         (list of) int of subject(s) to be fetched
-
+    dataset_kwargs: dict, optional
+        optional dictionary containing keyword arguments
+        to pass to the moabb dataset when instantiating it.
```
Comment (agramfort) [inline braindecode/datasets/moabb.py:81]:
```suggestion
        to pass to the moabb dataset when instantiating it.

```

**Review by agramfort — APPROVED** [review]
just a nitpick !

thx @PierreGtch

**Comment by agramfort** [comment]
@robintibor @gemeinl merge if happy

thx @PierreGtch

**Pull request merged**